addons/prometheus/prometheus.yaml

```diff
@@ -230,13 +230,13 @@ spec:
                 scheme: http
         prometheusSpec:
           image:
-            tag: v2.19.2
+            tag: v2.22.1
```

Comment:
we can probably omit this. the chart defaults the image to 2.22.1 so we don't need to overwrite it here unless we needed to upgrade. the risk of leaving it here is that we may accidentally overwrite a new default in the next upgrade